src/App/App.js

```diff
@@ -53,22 +55,34 @@ function App({ dexieCurrentUserInstance }) {
     Object.entries(projectsWithSyncErrors).forEach((projectWithSyncErrorsEntry) => {
       const projectId = projectWithSyncErrorsEntry[0]
       const { name: projectName, apiDataTablesThatRejectedSyncing } = projectWithSyncErrorsEntry[1]
-
-      const syncErrorUserMessaging = (
-        <div data-testid={`sync-error-for-project-${projectId}`}>
-          <P>{language.error.getPushSyncErrorMessage(projectName)}</P>
-          {language.error.pushSyncErrorMessageUnsavedData}
-          <ul>
-            {apiDataTablesThatRejectedSyncing?.map((rejectedDataTableName) => (
-              <li key={rejectedDataTableName}>
-                {language.apiDataTableNames[rejectedDataTableName]}
-              </li>
-            ))}
-          </ul>
-        </div>
-      )
-
-      toast.error(...getToastArguments(syncErrorUserMessaging))
+      //const currentUrlPath = window.location.href
+      // the test environment wont see client side routing paths (I cosoled it and looked at the console output of the test)
+      // console.log(currentUrlPath)
+      console.log(location.pathname, projectId, location.pathname.includes(projectId))
+
+      //const isErrorSpecificToProject = location.pathname.includes(projectId)
+      // the projectId here isnt necessarily the same one as in the url since we have offline ready projects which will be pushed/synced no matter what project page you are on.
+      // instead we can check if there is a project id.
+
+      const isCurrentRouteAProjectPage = !!getProjectIdFromLocation(location)
+
+      if (isCurrentRouteAProjectPage) {
```

Comment:
will this be for any project match though? what about matching the ids

Reply:
ya, we dont care what project the toast if for, just dont show it on the projects list page. actually let me read the ticket
...

Reply:
k, the ticket technically says only on project related pages, but I think you bring up a good point. @saanobhaai if you are on a project A page, do you only want to see push errors for project A even if projectB is producing errors?

Reply:
exactly, that was my understanding. only show project toasts to the relevant project you are on

Reply:
I wasnt thinking when I simplified....